homeassistant/components/poolstation/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for PoolStation integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import ClientResponseError, DummyCookieJar
+from pypoolstation import Account, AuthenticationException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DOMAIN, TOKEN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Poolstation."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str]
+
+        if user_input is None:
+            return self.async_show_form(step_id="user", data_schema=DATA_SCHEMA)
+        errors = {}
+        session = async_create_clientsession(self.hass, cookie_jar=DummyCookieJar)
```

Comment:
DummyCookieJar throws the cookies away, is that wanted here?

Reply:
Yes, it's 100% necessary.